server/__test__/rest/user.test.ts

```diff
@@ -1,13 +1,16 @@
-const request = require('supertest');
-const app = require('../../app');
-const mongoose = require('mongoose');
-const { settings, dbURI } = require('../../config/db');
-const User = require('../../models/User');
-const faker = require('faker');
+import request, { Response } from 'supertest';
+import { app } from '../../app';
+import mongoose from 'mongoose';
+import { settings, dbURI } from '../../config/db';
+import User from '../../models/User';
+import faker from 'faker';
+import { ITestUser } from '../types/user';
 
 describe('some initial tests for user', () => {
 	// global variables that we can use in all tests
-	let randomUser, registerResponse, connection;
+	let connection: typeof mongoose;
+	let randomUser: ITestUser;
```

Comment:
You will never change value of this variable. Why this is declare by let?

Reply:
I think yes, that name highlights that this user is only a set of random valid data and will be different every run. But of someone figure better name I will gladly change :)